tests/test-tokenizer-random.py

```diff
@@ -458,8 +458,8 @@ def check_detokenizer(text: str, text1: str, text2: str) -> bool:
             i = find_first_mismatch(ids1, ids2)
             ids1 = list(ids1)[max(0, i - 2) : i + 5 + 1]
             ids2 = list(ids2)[max(0, i - 2) : i + 5 + 1]
-            logger.error(" Expected: " + str(ids1))
-            logger.error("   Result: " + str(ids2))
+            logger.error(" Expected: " + str(ids1) + f" {[tokenizer1.decode([id]) for id in ids1]}")
+            logger.error("   Result: " + str(ids2) + f" {[tokenizer2.decode([id]) for id in ids2]}")
```

Comment:
I've reverted these two lines in <https://github.com/ggerganov/llama.cpp/pull/8228/commits/59ce85318aeace3d100395b86061fd10b3af6f54>, even though they were useful for debugging, because this is a test script anyway, and because these lines also conflicts with #8379